.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -23,6 +23,12 @@ COPY . .
 ENV DOCKER_GAPIC_GENERATOR_VERSION="2.55.2-SNAPSHOT"
 # {x-version-update-end}
 
+# Download the java formatter
+RUN mvn -pl gapic-generator-java-pom-parent help:evaluate -Dexpression='google-java-format.version' -q -DforceStdout > /java-formatter-version
+RUN cat /java-formatter-version
+RUN curl -o "/google-java-format.jar" "https://maven-central.storage-download.googleapis.com/maven2/com/google/googlejavaformat/google-java-format/$(cat /java-formatter-version)/google-java-format-$(cat /java-formatter-version)-all-deps.jar"
```

Comment:
```suggestion
RUN V=$(cat /java-formatter-version) curl -o "/google-java-format.jar" "https://maven-central.storage-download.googleapis.com/maven2/com/google/googlejavaformat/google-java-format/${V}/google-java-format-${V}-all-deps.jar"
```

Reply:
Nice

Reply:
Seems like the form `X=$(command1) echo "${X}"` needs `&&` or will otherwise evaluate `${X}` as an empty string.
This should be `X=$(command1) && echo "${command1}"` (I found by trial and error).